src/evaluation/searcher.h

```diff
@@ -454,8 +449,6 @@ class Searcher {
 
     static inline uint8_t s_numSearchers {};
 
-    static inline std::atomic_bool s_searchStopped { true };
```

Comment:
This flag will probably have to be re-introduced for lazySMP, because searches can be stopped for reasons other than a timeout.

I think there's a discussion to be had here about what's cleanest - adding and removing components, or leaving them in for fewer changes across the history.

Reply:
With the change from above, I can reintroduce the flag.
In general I prefer to have each commit being independent. They should always compile and not have logical errors.
That said I think this is a minor, so I'll just introduce it in the check from above - then it should play nicely with your branch :)